lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -140,19 +142,19 @@
     state: rebooted
 
 """
-
-import sys
-import time
-
+from time import sleep
+# import module snippets
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import ec2_argument_spec, get_aws_connection_info, boto3_conn
 try:
-    import boto
-    from boto.elasticache import connect_to_region
-    HAS_BOTO = True
+    import botocore
+    HAS_BOTO3 = True
```

Comment:
not quite sure which of the lines can be deleted and why. Could you explain or point me to some helpful documentation?
The botocore is needed and HAS_BOTO3 is used in all the examples.